.gitlab-ci.yml

```diff
@@ -10,15 +10,15 @@ before_script:
   # Enable GitLab registry
   - docker login -u gitlab-ci-token -p ${CI_BUILD_TOKEN} ${REGISTRY}
 
-.force_build_template: &force_build_definition
-  only:
-    - /^.*-force-build$/
-
 .build_template: &build_definition
   stage: build
+  tags:
+    - deploy_test
   script:
     - >
-      if  ${CHECK_GIT} | grep "^versions/${OS}_${VER}$" || \
+      if  echo ${CI_COMMIT_REF_NAME} | grep "master" || \
+          echo ${CI_COMMIT_REF_NAME} | grep "^.*-force-build$" || \
```

Comment:
I don't think we should exclude so named branches.

Reply:
Right, seems it is not needed.